actgen/train.py

```diff
@@ -104,13 +106,22 @@ def setup(self):
 
         if self.params['oracle'] and not self.params['dqn_train_pin_other_q_values']:
             raise RuntimeError('dqn_train_pin_other_q_values must be set to true when performing oracle action generalization')
+        
+        if self.params['gpu']:
+            if torch.cuda.is_available():
+                torch.backends.cudnn.benchmark = True
+                self.params['device'] = torch.device('cuda')
+            else:
+                raise RuntimeError('there is no GPU available')
+        else:
+            self.params['device'] = torch.device('cpu')
```

Comment:
I'm seeing the same code in at least 3 places. While you're changing it to `--disable-gpu`, can we also refactor and throw this block in utils?